chimney/src/test/scala/io/scalaland/chimney/DslSpec.scala

```diff
@@ -596,7 +596,7 @@ object DslSpec extends TestSuite {
       "method is disabled by default" - {
         compileError("""Foobar("param").into[Foobar3].transform""").check(
           "",
-          "method: java.lang.String - no accessor named method in source type io.scalaland.chimney.DslSpec.Foobar"
+          "method: java.lang.String - no accessor named method in source type io.scalaland.chimney.DslSpec.Foobar, but there was a method with this name. Try to `.enableMethodAccessors`"
```

Comment:
Now that I think about it, there migh be more cases when we would provide some hints to the user. In that case should we always provide a separate ADT just to trace things? Perhaps, we could have one common metadata, for storing these, which would be extensible? After all we use it only if compilation fails to generate more precise error message?

Not necessarily an issue for now, but I would like to discuss this for the future.